File: miraculous-frontend/miraculous-ui/src/dto/genre-dto/genre-dto.ts
export class GenreDto {

  id: number;
  title: string;
  genrePictureUrlName: string;
  totalNumberOfLikes: number;
  description: string;


  constructor(id?: number, title?: string, genrePictureUrl?: string, totalNumberOfLikes?: number, description?: string) {
    this.id = id;
    this.title = title;
    this.genrePictureUrlName = genrePictureUrl;
    this.totalNumberOfLikes = totalNumberOfLikes;
    this.description = description;
  }
}

File: miraculous-backend/src/main/java/com/example/miraculousbackend/dto/GenreDto.java
package com.example.miraculousbackend.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GenreDto {

    private Long id;
    private String title;
    private String genrePictureUrlName;
    private Long totalNumberOfLikes;
    private String description;
}
